ai/gpt_finetune/data_process.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+import json
+from utils import dicts_to_jsonl, parse_cfg_file
+
+default_sys_prompt = "You are the assistant that helps classifying issues in Antrea, a software providing networking and security services for a Kubernetes cluster. Please only output the classification results as some words, which are closed by apostrophes and connected by comma."
+
+parser = argparse.ArgumentParser(description='This script will process the json data to the right jsonl format.')
+parser.add_argument('-i', '--input', type=str, help='Input json file path')
+parser.add_argument('-o', '--output', type=str, help='Output jsonl file path')
+parser.add_argument('-c', '--config', type=str, help='Data processing configuration')
+args = parser.parse_args()
+
+input_file = args.input
+output_file = args.output
+
+if args.config:
+    cfg_dict = parse_cfg_file(args.config)
+    if 'sys_prompt' in cfg_dict:
+        sys_prompt = cfg_dict['sys_prompt']
+    else:
+        sys_prompt = default_sys_prompt
+    if 'task_type' in cfg_dict:
+        task_type = cfg_dict['task_type']
+    else:
+        task_type = "answering"
+else:
+    sys_prompt = default_sys_prompt
+    task_type = "answering"
```

Comment:
Can you try if this change works?
```suggestion
sys_prompt, task_type = DEFAULT_SYS_PROMPT, DEFAULT_TASK_TYPE
if args.config:
    cfg_dict = parse_cfg_file(args.config)
    sys_prompt = cfg_dict.get('sys_prompt', default_sys_prompt)
    task_type = cfg_dict.get('task_type', 'answering')
```

Reply:
updated